README.rst

```diff
@@ -2,10 +2,10 @@
 matplotlib
 ##########
 
-matplotlib is a python 2D plotting library which produces publication
-quality figures in a variety of hardcopy formats and interactive
-environments across platforms. matplotlib can be used in python
-scripts, the python and ipython shell (ala matlab or mathematica), web
+matplotlib is a Python 2D plotting library which produces publication-quality
```

Comment:
There was discussion of this on the mailing list, but I do not thing there was a conclusive answer.